cmd/controlplane/management_controller.go

```diff
@@ -17,6 +17,7 @@ import (
 
 	kargoapi "github.com/akuity/kargo/api/v1alpha1"
 	"github.com/akuity/kargo/internal/controller/management/namespaces"
+	projectconfigs "github.com/akuity/kargo/internal/controller/management/project-configs"
```

Comment:
I would suggest making the path `internal/controller/management/projectconfigs` so the issue caused by the dash just doesn't even come up.